pkg/storage/internalstorage/config.go

```diff
@@ -145,7 +146,18 @@ func (cfg *Config) getConnPoolConfig() (ConnPoolConfig, error) {
 
 func (cfg *Config) genMySQLConfig() (*mysql.Config, error) {
 	if cfg.DSN != "" {
-		return mysql.ParseDSN(cfg.DSN)
+		mysqlConfig, err := mysql.ParseDSN(cfg.DSN)
+		if err != nil {
+			return nil, err
+		}
+		if mysqlConfig.Passwd == "" {
+			mysqlConfig.Passwd = os.Getenv(databasePasswordEnvName)
+		}
+		if !mysqlConfig.ParseTime {
+			klog.Warningln("Mysql query param parseTime=false has been ignored, and set to true")
```

Comment:
The default value of `mysqlConfig.ParseTime` is false, so we can skip warning or check from `cfs.DSN` string